src/renderer/components/stake/add/AddStake.tsx

```diff
@@ -198,13 +216,16 @@ export const AddStake: React.FC<Props> = ({
           />
           <Row align="middle">
             <Col>
-              <Styled.ReloadFeeButton onClick={reloadFee} disabled={RD.isPending(feeRD)}>
+              <Styled.ReloadFeeButton onClick={reloadFees} disabled={RD.isPending(fees)}>
                 <SyncOutlined />
               </Styled.ReloadFeeButton>
             </Col>
             <Col>
-              <Styled.FeeLabel disabled={RD.isPending(feeRD)}>
-                {intl.formatMessage({ id: 'common.fee' })}: {feeLabel}
+              <Styled.FeeLabel disabled={RD.isPending(fees)}>
```

Comment:
maybe we could use `loading` prop here ?

Reply:
We would lost `Fee:` text before fee amounts then.